programs/relayer/src/modules/cosmos_to_eth.rs

```diff
@@ -118,8 +116,10 @@ impl RelayerService for CosmosToEthRelayerModuleServer {
         &self,
         request: Request<api::RelayByTxRequest>,
     ) -> Result<Response<api::RelayByTxResponse>, tonic::Status> {
-        tracing::info!("Handling relay by tx request...");
+        tracing::info!("Handling relay by tx request for cosmos to eth...");
         let inner_req = request.into_inner();
+        tracing::info!("Got {} source_tx_ids", inner_req.source_tx_ids.len());
+        tracing::info!("Got {} timeout_tx_ids", inner_req.timeout_tx_ids.len());
```

Comment:
does this format match our logs in other relayer modules? It would be nice to use the same format

Reply:
More or less, but the other ones didn't have this logging, so changing them a little and adding it to those as well for consistency.